src/Directory.Build.targets

```diff
@@ -13,10 +13,21 @@
     <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);IncludePDBsInPackage</TargetsForTfmSpecificContentInPackage>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
FYI in my followup PR to move to Particular.Packaging, the need for this Directory.Build.targets file is going away, so these new items would be better in the main csproj instead. 

However, I don't mind doing that as part of my PR, since they are fine where they are for this one.